src/txgraph.cpp

```diff
@@ -1449,6 +1465,7 @@ Cluster::Cluster(TxGraphImpl& graph, const FeeFrac& feerate, GraphIndex graph_in
 TxGraph::Ref TxGraphImpl::AddTransaction(const FeeFrac& feerate) noexcept
 {
     Assume(m_chunkindex_observers == 0 || m_clustersets.size() > 1);
+    Assume(feerate.size > 0 && uint64_t(feerate.size) <= m_max_cluster_size);
```

Comment:
FYI -- in my rebase of #28676, I'm seeing `tx_pool` fuzz test failures due to this line. Not clear to me whether we should require the caller to enforce the policy requirement that a single tx be below the cluster size limit, or just let the caller discover a changeset is oversized and then reject?